### PR TITLE
Update perfusion and fixation techniques

### PR DESCRIPTION
**Pull request by MaaikevS**
Related to issue #70

### CODE REVIEW

**Comment by MaaikevS** [comment]
@UlrikeS91 @lzehl Could either of you review these changes?
I will delete "tissue fixation technique" when it is no longer linked to any instances in the KGE

**Comment by UlrikeS91** [comment]
You forgot to delete "tissue fixation technique". I'll review the rest in the meantime 😉

File: instances/technique/perfusionFixationTechnique.jsonld [hunk 1]
```diff
@@ -4,7 +4,7 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/perfusionFixationTechnique",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
+  "definition": "Perfusion fixation is a technique that uses the vascular system to distribute fixatives throughout tissue.",
```
Comment (UlrikeS91) [inline instances/technique/perfusionFixationTechnique.jsonld]: Maybe align this more with the definition for "diffusion fixation"?

Something like:
"Perfusion fixation is a fixation technique to preserve specimen permanently as faithfully as possible compared to the living state by using the vascular system to distribute fixatives throughout the tissue." 

But feel free to adjust further :)

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

**Comment by UlrikeS91** [comment]
> @UlrikeS91 @lzehl Could either of you review these changes? I will delete "tissue fixation technique" when it is no longer linked to any instances in the KGE

Then ignore my first comment 😆 

I would adjust one of the definitions. The rest looks fine to me.

**Comment by MaaikevS** [comment]
@UlrikeS91 Ok, tissue fixation technique is now also deleted and the definition of perfusion fixation technique is changed to what you proposed

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**